planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -389,8 +392,42 @@ class MoveGroupWrapper : protected py_bindings_tools::ROScppInitializer,
      return constraints_str;
   }
 
+  std::string retimeTrajectory(const std::string& ref_state_str,
```

Comment:
I haven't seen this called "retime" anywhere else in the code, can we just call it "parameterizeTrajectory" to keep it consistent with naming convention? And add the term "retime" in the comments.